config/settings_schema.json

```diff
@@ -33,6 +33,38 @@
       }
     ]
   },
+  {
+    "name": "t:settings_schema.brand_information.name",
+    "settings": [
+      {
+        "type": "richtext",
+        "id": "brand_headline",
+        "default": "<p></p>",
```

Comment:
I wonder if we should have something here 🤔 I guess we don't want to create work for people to go and delete it. 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/02-40-3x4ru-bp5a6.png)
</details>

Reply:
We shouldn't use `<p></p>` as the default value here, as this results in invalid HTML. A `p` element can't be a child of a heading.
![invalid](https://user-images.githubusercontent.com/22390758/205589654-2d5f65e4-858a-48ee-b731-75dd65a40074.png)